### PR TITLE
Feature: Choose a license type when creating a new file

### PR DESCRIPTION
**Pull request by ibraimgm**
Created a new license type `choose` that, if selected, aloows the user to select the desired license type from a list. Demo of the final result:

licenser

**IMPORTANT:** Since I don't really know typescript or any typescript/vscode best practices, I tried to do the simplest thing that works. Feedback will be appreciated.

### CODE REVIEW

**Comment by ibraimgm** [comment]
@ymotongpoo Agreed, `choose` may not be descriptive enough. Updated it to `Choose from list`, let me know if there is any other problem.

File: src/extension.ts [hunk 1]
```diff
@@ -99,10 +132,27 @@ class Licenser {
...
+            }
+        });
+    }
+
+    private _chooseLicenseType(): Thenable<string> {
         let licenserSetting = vscode.workspace.getConfiguration("licenser");
         let licenseType = licenserSetting.get<string>("license");
-        const license = this.getLicense(licenseType);
 
+        if (isNullOrUndefined(licenseType) || licenseType.toLowerCase() == "choose from list") {
```
Comment (ymotongpoo) [inline src/extension.ts]: Can you make the string "choose from list" as `const` value in this module or class just in case?

Comment (ibraimgm) [reply src/extension.ts]: Done!

**Comment by ibraimgm** [comment]
Done. Changed it to a constant and rebased.

**Comment by ymotongpoo** [comment]
@ibraimgm I appreciate your contribution. merged!

**Pull request merged**